MDS_3/examples/MDS_3/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+# Created by the script cgal_create_CMakeLists
+# This is the CMake script for compiling a set of CGAL applications.
+
+cmake_minimum_required(VERSION 3.1...3.14)
+
+project(MDS_3_Examples)
+
+# CGAL and its components
+find_package(CGAL REQUIRED)
+
+# Boost and its components
+find_package(Boost REQUIRED)
+if(NOT Boost_FOUND)
+  message(
+    STATUS "This project requires the Boost library, and will not be compiled.")
+  return()
+endif()
```

Comment:
This is useless because `find_package(Boost)` has the `REQUIRED` keyword.
```suggestion
```

Actually the line `find_package(Boost REQUIRED)` itself is useless, because `CGALConfig.cmake` will search for Boost header as well.